src/ReportGenerator.Core/Reporting/Builders/CloverReportBuilder.cs

```diff
@@ -386,7 +388,10 @@ public void CreateSummaryReport(SummaryResult summaryResult)
 
             Logger.InfoFormat(Resources.WritingReportFile, targetPath);
 
-            result.Save(targetPath);
+            using (var writer = new XmlTextWriter(targetPath, new UTF8Encoding(encoderShouldEmitUTF8Identifier: false)))
```

Comment:
I am not super familiar with the project layout. Is there a good place to cache this `UTF8Encoding` so that I can use it everywhere without having to create it repeatedly each time?